src/graph/graph-state-manager.ts

```diff
@@ -20,30 +20,20 @@ import { ConfigService } from '../config/config.service';
 
 @Injectable()
 export class GraphStateManager implements OnApplicationBootstrap {
-  private graphState: Graph;
-
   private environment: EnvironmentInterface; // Environment details
 
   private schemaIds: { [key: string]: { [key: string]: number } };
 
   private graphKeySchemaId: number;
 
-  private static graphStateFinalizer = new FinalizationRegistry((graphState: Graph) => {
-    if (graphState) {
-      graphState.freeGraphState();
-    }
-  });
-
   public onApplicationBootstrap() {
-    if (!this.graphState) {
-      throw new Error('Unable to initialize schema ids');
-    }
+    const graphState = this.getGraphState();
 
-    const publicFollow = this.graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Follow, PrivacyType.Public);
-    const privateFollow = this.graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Follow, PrivacyType.Private);
-    const privateFriend = this.graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Friendship, PrivacyType.Private);
+    const publicFollow = graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Follow, PrivacyType.Public);
+    const privateFollow = graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Follow, PrivacyType.Private);
+    const privateFriend = graphState.getSchemaIdFromConfig(this.environment, ConnectionType.Friendship, PrivacyType.Private);
```

Comment:
NB, hopefully this can all go away soon--once Mainnet is updated to support schema names, we can eliminate the schema IDs from the graph config and just look them up on-chain by name.